planning/behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -848,6 +848,25 @@ std::optional<std::pair<PullOverPath, GoalCandidate>> GoalPlannerModule::selectP
   const std::vector<PullOverPath> & pull_over_path_candidates,
   const GoalCandidates & goal_candidates, const double collision_check_margin) const
 {
+  const auto & goal_pose = planner_data_->route_handler->getOriginalGoalPose();
+  const double backward_length =
+    parameters_->backward_goal_search_length + parameters_->decide_path_distance;
+  const auto & prev_module_output_path = getPreviousModuleOutput().path;
+  const double prev_path_back_to_goal_dist = calcSignedArcLength(
+    prev_module_output_path.points, prev_module_output_path.points.back().point.pose.position,
+    goal_pose.position);
```

Comment:
prev_module_output_path.points.front?

Reply:
fixed in f7a10caf9b2fdc9118104eb9d548ff0e56fec1af